example/service.twirp.go

```diff
@@ -45,24 +45,31 @@ type Haberdasher interface {
 type haberdasherProtobufClient struct {
 	client HTTPClient
 	urls   [1]string
+	opts   twirp.ClientOptions
 }
 
 // NewHaberdasherProtobufClient creates a Protobuf client that implements the Haberdasher interface.
 // It communicates using Protobuf and can be configured with a custom HTTPClient.
-func NewHaberdasherProtobufClient(addr string, client HTTPClient) Haberdasher {
+func NewHaberdasherProtobufClient(addr string, client HTTPClient, opt ...twirp.ClientOption) Haberdasher {
```

Comment:
Please name the argument `opts` instead of `opt`, to match its name within the struct (and the fact that it is variadic).